### DISCUSSION TITLE
F-Droid app

### DISCUSSION THREAD

**Discussion by nuraiman** [emericg/WatchFlower#61] [Q&A] [answered]
Hi, 
WatchFlower is very nice!!
Do you plan also to add it to F-Droid?
https://f-droid.org/docs/Inclusion_Policy/

**Comment by emericg** [comment, marked as answer]
Well thank you :)

Honestly it's not that I'm completely opposed to it, but it's already way too much work to publish this on the 5 platforms and 2 stores already supported... Also I know nothing about F-Droid.
In the future I want to automate store uploads to streamline publications, but I'm sure F-Droid would need an entire kind of integration anyway.
